hugegraph-rocksdb/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>org.rocksdb</groupId>
             <artifactId>rocksdbjni</artifactId>
-            <version>6.10.2</version>
+            <version>6.22.1.1</version>
```

Comment:
TODO: bump up to 6.30 when it's released (now 6.27.3 available, but not supports auto prefix mode and memtable whole key filtering)